docs/pbc/all/business-intelligence/202410.0/amazon-quicksight-third-party-integration/set-up-amazon-quicksight.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Set up Amazon QuickSight
+description: First steps to set up Amazon QuickSight
+template: concept-topic-template
+last_updated: Oct 24, 2024
+---
+
+This document describes how to set up Amazon QuickSight.
+
+1. [Install the Amazon QuickSight feature]().
+  At this stage, the **Analytics** section is displayed in the Back Office, but the functionality is not available.
```

Comment:
please remove it: "At this stage, the Analytics section is displayed in the Back Office, but the functionality is not available." and make this point number 2. The first should be "To request Business Intelligence powered by Amazon QuickSight…"